### ISSUE TITLE
Use host.docker.internal:port instead of network_mode: host In Docker Compose

### ISSUE THREAD

**Issue by FlattusBlastus** [nextcloud/all-in-one#3502]
Community Containers:
As noted, network_mode=host doesnt work in Windows Docker Desktop

### Describe the solution you'd like
host.docker.internal:port is the official solution from Docker.
Works in Linux and Windows.

**Comment by FlattusBlastus** [comment]
What I've done in the past is in Compose, just use your ports like normal but leave network_mode out.
In your reverse proxy, use the host.docker.internal:<your port you used in Compose> and that works.

**Comment by FlattusBlastus** [comment]
Simon, I will make myself available for a work session if wanted.

**Comment by Zoey2936** [comment]
https://docs.docker.com/desktop/networking/#i-want-to-connect-from-a-container-to-a-service-on-the-host If I read this, it seems like it is something totally different (the sense of network mode host is to NOT have port mappings)

**Comment by FlattusBlastus** [comment]
@Zoey2936 Indeed. You are absolutely right. However, its what's available on the underlying platform.: https://docs.docker.com/desktop/networking/#use-cases-and-workarounds-for-all-platforms

**Comment by Zoey2936** [comment]
Do you have use cases for AIO where this could be used?
